_scripts/check-db-patcher.sh

```diff
@@ -2,24 +2,30 @@
 NAME="patcher.mjs" # nodejs script's name here
 RETRY=60
 
-# Wait for $NAME script to start
-for i in $(eval echo "{1..$RETRY}"); do
-  if [[ $(pgrep -f $NAME) == "" ]]; then
-    sleep 1
-  else
-    break;
-  fi
-done
+echo -e "\nChecking for DB patches..."
 
-# Wait for $NAME script to finish
-for j in $(eval echo "{1..$RETRY}"); do
-  if [[ $(pgrep -f $NAME) == "" ]]; then
-    exit 0
+# Look for a process matching the script name
+PATCHER_PID=$(pgrep -f "$NAME")
+
+if [[ -z "$PATCHER_PID" ]]; then
+  echo "✅ No DB patches needed"
+  exit 0
+fi
+
+# Confirm the process is running
+if ! ps -p "$PATCHER_PID" > /dev/null; then
```

Comment:
Brackets should not be needed here - Bash directly evaluates the exit condition of the command